eessi_bot_job_manager.py

```diff
@@ -313,21 +337,34 @@ def process_new_job(self, new_job):
             symlink_source = os.path.join(self.submitted_jobs_dir, job_id)
             log(
                 "process_new_job(): create a symlink: %s -> %s"
-                % (symlink_source, match.group(1)),
+                % (symlink_source, job_info['WorkDir']),
                 self.logfile,
             )
-            os.symlink(match.group(1), symlink_source)
-
-            release_cmd = "%s release %s" % (
-                self.scontrol_command,
-                job_id,
-            )
+            os.symlink(job_info['WorkDir'], symlink_source)
+
+            # handle different job handover protocols
+            #   *_HOLD_RELEASE: job was submitted with '--hold' and shall be
+            #                   released with 'scontrol release JOB_ID'
+            #   *_DELAYED_BEGIN: job was submitted with '--begin=now+SOMEDELAY',
+            #                   no extra action is needed
+            job_status = ''
+            extra_info = ''
+            if self.job_handover_protocol == config.JOB_HANDOVER_HOLD_RELEASE:
+                release_cmd = "%s release %s" % (
+                    self.scontrol_command,
+                    job_id,
+                )
 
-            release_output, release_err, release_exitcode = run_cmd(
-                release_cmd,
-                "process_new_job(): scontrol command",
-                log_file=self.logfile,
-            )
+                release_output, release_err, release_exitcode = run_cmd(
+                    release_cmd,
+                    "process_new_job(): scontrol command",
+                    log_file=self.logfile,
+                )
+                job_status = 'released'
+                extra_info = ''
+            elif self.job_handover_protocol == config.JOB_HANDOVER_DELAYED_BEGIN:
```

Comment:
Similar, I think this should be
```suggestion
            elif self.job_handover_protocol == config.JOB_HANDOVER_PROTOCOL_HOLD_RELEASE:
```

Reply:
Fixed in https://github.com/EESSI/eessi-bot-software-layer/pull/297/commits/782a86235fcb9757b38a0a7dc339edf61320f184